src/server/kiwix-serve.cpp

```diff
@@ -393,13 +383,12 @@ int main(int argc, char** argv)
   if (! server.start()) {
     exit(1);
   }
-
-  std::string prefix = "http://";
-  kiwix::IpAddress addresses = server.getAddress();
-  std::string suffix = ":" + std::to_string(server.getPort()) + normalizeRootUrl(rootLocation);
+  
   std::cout << "The Kiwix server is running and can be accessed in the local network at: " << std::endl;
-  if(!addresses.addr.empty()) std::cout << "  - " << prefix << addresses.addr << suffix << std::endl;
-  if(!addresses.addr6.empty()) std::cout << "  - " << prefix << "[" << addresses.addr6 << "]" <<  suffix << std::endl;
+  const std::vector<std::string> serverAccessUrls = server.getServerAccessUrls();
```

Comment:
The variable is not justified. Please call the function right in the `for` loop below